docker-compose.yml

```
@@ -0,0 +1,26 @@
+version: "3"
```

Acho bom especificar a versão minor do docker-compose também:
```suggestion
version: "3.8"
```
Além disso sugiro usar sempre a ultima versão estável (se possível)

O que acha ?
Ref: https://docs.docker.com/compose/compose-file/compose-versioning/